packages/slice-machine/src/modules/availableCustomTypes/factory.ts

```diff
@@ -1,35 +1,74 @@
-import { CustomTypeSM, TabFields } from "@lib/models/common/CustomType";
+import { CustomTypeSM, TabSM } from "@lib/models/common/CustomType";
+
+const DEFAULT_SEO_TAB: TabSM = {
+  key: "Metadata",
+  value: [
+    {
+      key: "meta_title",
+      value: {
+        type: "Text",
+        config: {
+          label: "Meta Title",
+          placeholder:
+            "A title of the page used for social media and search engines",
+        },
+      },
+    },
+    {
+      key: "meta_description",
+      value: {
+        type: "StructuredText",
+        config: {
+          label: "Meta Description",
+          placeholder: "A brief summary of the page",
+        },
+      },
+    },
+    {
+      key: "meta_image",
+      value: {
+        type: "Image",
+        config: {
+          label: "Meta Image",
+          constraint: {
+            width: 2400,
+            height: 1260,
+          },
+        },
+      },
+    },
+  ],
+};
 
 export const createCustomType = (
```

Comment:
We need to add the condition for format "page" or "custom" here. Your work is only for Page types not Custom types. With new Prismic types internal 2.0.0.beta.1, you should be able to have the condition.

Then you can property manage it. Sorry if the issue was not clear